ci-operator/jobs/openshift/release/openshift-release-periodics.yaml

```diff
@@ -134,3 +134,28 @@ periodics:
         requests:
           cpu: 10m
     serviceAccountName: config-updater
+- agent: kubernetes
+  interval: 12h
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  decorate: true
+  name: openshift-release-master-config-bootstrapper
+  spec:
+    containers:
+    - name: ""
+      image: gcr.io/k8s-prow/config-bootstrapper:latest
```

Comment:
Please use the tag that matches the rest of Prow.

Reply:
addressed